servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpExecutionStrategyInfluencer.java

```diff
@@ -47,7 +56,8 @@ default HttpExecutionStrategy influenceStrategy(HttpExecutionStrategy strategy)
     @Override
     default HttpExecutionStrategy requiredOffloads() {
         // safe default--implementations are expected to override
-        return HttpExecutionStrategies.offloadAll();
+        HttpExecutionStrategy result = influenceStrategy(defaultStrategy());
+        return defaultStrategy() == result ? offloadAll() : result;
```

Comment:
0.41.x users who want to say "my filter does not block" implement `influenceStrategy` this way:
```java

    @Override
    public HttpExecutionStrategy influenceStrategy(HttpExecutionStrategy strategy) {
        // No influence since we do not block.
        return strategy;
    }
```
it will always return `defaultStrategy()` and result in `offloadAll()` instead of `offloadNone()`

Reply:
You are correct. I've addressed this issue and added two more test cases.